using-valtimo/process/systemprocesses.md

```diff
@@ -0,0 +1,31 @@
+# System processes
+
+System processes are a special kind of process. A system process describes an often used process 
+that is used by the system. These process should be handled with care so when a process
+is a system process it will be read-only by default. When a process is read-only it is not possible
+to update the process or deploy it.
+
+![Read only system process](img/read-only-system-process.png)
+
+A process can be marked as a system process by setting a property called `systemProcess`
+to the value `true` on the model itself under the `Extensions` tab
```

Comment:
`the value` can be omitted here.

Reply:
Also missing punctuation at the end of the sentence.

Reply:
You mean that exact text only?